packages/buildpacks/package.json

```diff
@@ -5,22 +5,19 @@
   "bugs": "https://github.com/heroku/cli/issues",
   "dependencies": {
     "@heroku-cli/color": "^1.1.14",
-    "@heroku-cli/command": "^8.4.1",
+    "@heroku-cli/command": "^9.0.0",
     "@heroku/buildpack-registry": "^1.0.1",
-    "@oclif/config": "^1.12.10",
-    "@oclif/plugin-legacy": "^1.2.0",
-    "cli-ux": "^4.9.3",
+    "@oclif/core": "^1.21.0",
     "heroku-cli-util": "^8.0.11",
     "http-call": "^5.2.3",
     "lodash": "^4.17.11",
     "true-myth": "2.2.3",
     "valid-url": "^1.0.9"
   },
   "devDependencies": {
-    "@fancy-test/nock": "^0.1.1",
     "@heroku-cli/dev-cli": "^1.26.13",
     "@oclif/plugin-help": "^2.1.6",
```

Comment:
dev-cli and plugin-help should also be able to be deleted, as these are now part of oclif/core